### ISSUE TITLE
[Discuss] Inline admonitions

### ISSUE THREAD

**Issue by bmorelli25** [elastic/docs-builder#108]
### Summary

You can find an example of an inline admonition in this page:

[Data set quality | Elastic Observability [8.17] | Elastic](https://www.elastic.co/guide/en/observability/current/monitor-datasets.html) 

The text [beta] on top of the page is an inline admonition. If you have over it, you will see a tooltip-style box. There is no equivalent functionality in MyST, so for now we are converting it an admonition of type ‘important'.

~~~
```{admonition} <title>
:class: important
<text>
```
~~~

### To do

* #94 and #95 will add support for beta/preview callouts to V3
* After these hit, determine a new mapping from asciidoc to v3 for inline admonitions
* Determine which inline admonitions exist outside of the context of versioning and also need to be added

**Comment by Mpdreamz** [comment]
I don't think we should support this for reasons mentioned here: https://github.com/elastic/docs-builder/issues/94#issuecomment-2547931900

**Comment by bmorelli25** [comment]
This decision to be validated during migration. Added as an open question to https://github.com/elastic/docs-builder/issues/160.

**Comment by bmorelli25** [comment]
From @siamakp-elastic in regards to inline admonitions in code callouts:

> Look at this page:
https://www.elastic.co/guide/en/observability/current/apm-open-telemetry-direct.html
The callout #8 has an inline admonition preview. I used to render them ('beta', 'preview', etc) as block admonitions. But admonition won't work in the callout case above. I've changed it to be just the title in the brackets: [preview].
That means that the documents that have an inline beta inline admonition, will just show [beta] without any more information. Note that the Asciidoc inline admonition acts like a hover tooltip.

**Comment by charlotte-hoblik** [comment]
I came across a couple of inline admonitions inside table cells, as shown in the screenshot. It is tricky to replace them using callouts or prose when only a few rows are marked in the table.

<img width="400" alt="Image" src="https://github.com/user-attachments/assets/498c497a-a445-4135-849f-225beba6e522" />
